libs/pulumi/src/executors/up/schema.d.ts

```diff
@@ -1,11 +1,22 @@
+export interface BuildTarget {
+    project: string
+    target: string
+    configuration?: string
+}
+
 export interface BuildExecutorSchema {
     targetProjectName: string
     configurationStackFormat?: string
-    buildTarget?: string
-    additionalBuildTargets?: Array<{
-        project: string
-        target: string
-        configuration?: string
-    }>
     infrastructureProject?: string
+    buildTargets?: Array<BuildTarget>
```

Comment:
Need to also update the JSON Schema file

Reply:
:drake-like: